### ISSUE TITLE
[Bug]: CloseButton and ClearButton don't support self-managed tooltips

### ISSUE THREAD

**Issue by spdev3000** [adobe/spectrum-web-components#4884]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

CloseButton, ClearButton

### Expected behavior

As normal ActionButton triggers a slotted (self-managed) tooltip - user would expect the same behavior for CloseButton and ClearButton 

### Actual behavior

CloseButton nor ClearButton shows slotted (self-managed) tooltip

### What browsers are you seeing the problem in?

Chrome

### How can we reproduce this issue?

1. Go to https://studio.webcomponents.dev/edit/4jQM84fJGHNtETZSNrFM/src/index.ts?p=stories
2. Hover over action-button => tooltip is showing
3. Hover over close button and clear button => no tooltip is showing

### Severity

SEV 4

**Comment by rubencarvalho** [comment]
I agree this is an inconsistency. While we triage the issue, could you elaborate on the use case where you’d need to have a tooltip on any of these buttons?

**Comment by spdev3000** [comment]
Sure, we are using the CloseButton in a custom dialog and the ClearButton in a search field. We'd like to show tooltips for both use cases.
